LBH.AdultSocialCare.Api/V1/Boundary/PayRuns/Request/DraftPayRunCreationRequest.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Threading.Tasks;
+using Common.AppConstants.Enums;
+using LBH.AdultSocialCare.Api.Attributes;
+using LBH.AdultSocialCare.Api.V1.AppConstants.Enums;
+using LBH.AdultSocialCare.Api.V1.Domain.Payments;
+using LBH.AdultSocialCare.Api.V1.Validations;
+
+namespace LBH.AdultSocialCare.Api.V1.Boundary.PayRuns.Request
+{
+    [GenerateMappingFor(typeof(DraftPayRunCreationDomain))]
+    public class DraftPayRunCreationRequest
+    {
+        [Required]
+        [Range(1, 3)]
```

Comment:
No need to use explicit validation for enum range, all enums are validated by EnumRangeValidator